docs/source/contrib_docs/doc-new-translation.md

```diff
@@ -0,0 +1,351 @@
+# Supporting translations of documentation
+
+We support and encourage the translation of Jupyter documentation to other languages as one way of
+making our community more inclusive and diverse. We are working toward having a consistent model for
+translation of [Sphinx](https://www.sphinx-doc.org/) documentation across Jupyter projects based on
+prior work in the [Python](https://python.org) and [Django](https://www.djangoproject.com/)
+communities. This project (https://jupyter.readthedocs.io) and the
+[Jupyter Docker Stacks project](https://jupyter-docker-stacks.readthedocs.io/) are early adopters,
+meant to prove out the workflows described on this page.
+
+## Overview
+
+After initial project setup, changes to Sphinx documentation and its translations follow a
+continuous integration (CI) and continuous deployment (CD), much like project source code. This flow
+has a handful of actors:
+
+- A person who makes changes to the English project documentation
+- A person who translates snippets of text in the English documentations into another language
+  (locale)
+- A continuous integration system like TravisCI, CircleCI, GitHub Actions
+- [ReadTheDocs](https:/readthedocs.org), our preferred service for building and hosting
+  documentation
+- [Transifex](https://transifex.com), a localization platform with free plans for open source
+  projects, a friendly web interface, and support for the defacto
+  [portable object (`.po`)](https://en.wikipedia.org/wiki/Gettext) translation format
+
+![Translation CI/CD](static/translation-ci-cd.png "Diagram of the translation continuous integration
```

Comment:
Updating to a newer version of recommonmark in `doc-requirements.txt` fixes the problem for me, along with many other warnings about the doc here.

![image](https://user-images.githubusercontent.com/153745/86313595-815ca180-bbf3-11ea-9b37-f751fc1220ad.png)

I'm going to unpin recommonmark and increase the minimum required sphinx version. We can see if that causes new issues in CI.

Reply:
I'd also be a fan of using the myst-parser but that's probably for another PR :-)

the diagram looks awesome! this is great! do you have a link to the source so that others could edit in the future?

Reply:
Thanks for the reminder about getting the source out of LucidChart into some format that we can update in the future. SVG is probably the best bet from available options. I'll add it to the static folder alongside the PNG for safe keeping.